week-3/implement/get-card-value.js

```diff
@@ -10,6 +10,70 @@
 // When the function getCardValue is called with this card string as input,
 // Then it should return the numerical card value
 
+function getCardValue(card, symbol) {
+  const NumberCards = ["2", "3", "4", "5", "6", "7", "8", "9"];
+  const cardSymbols = ["♠︎", "♣︎", "♥︎", "♦︎"];
+  const aceOfSpades = ["A"];
+  const faceCards = ["10", "J", "Q", "K"];
+  if (NumberCards.includes(card) && cardSymbols.includes(symbol)) {
```

Comment:
I think I intended for folks to use a single parameter here for the `card`. You've just 2 parameters which is also fine too! I think you can simplify this code though - does the card value depend on the card symbol (suit) in any way?